html/index.html

```diff
@@ -29,12 +35,193 @@
         <div class="icon-113-themify">
             <div class="ti-thumb-up"></div>
         </div>
+        <div class="better">Better</div>
+        <div class="faster">Faster</div>
+        <div class="together">Together</div>
+        <div class="lorem-ipsum-is-simpl">
+            Lorem ipsum is simply dummy text of the printing and type setting industry
+        </div>
         <div class="rectangle-2-copy">
             <div class="start-now">
                 START NOW
             </div>
-        </div>      
+        </div>
+    </div>
+
+    <!-- Second rectangle -->
+    <div class="mask">
+        <div class="today-i-learnt-image">
+            <img src="../images/Browser.png" alt="" srcset="">
+        </div>
+        <div class="rectangle-3">
+            <div class="icon-25-themify">
+                <div class="ti-bookmark-alt"></div>
+            </div>
+            <div class="today-i-learnt">Today I learnt</div>
+            <div class="lorem-ipsum-is-simpl-copy">
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry
+            </div>
+            <div class="lorem-ipsum-is-simpl-2">
```

Comment:
@linhlt311 em có thể dùng nhiều tag html hơn để sử dụng thay vì toàn bộ div

Reply:
Vâng ạ